### DISCUSSION TITLE
Add image loader for Storyblok

### DISCUSSION THREAD

**Discussion by florisdegraaff** [vercel/next.js#43813] [Ideas]
### Describe the feature you'd like to request

The next/image currently supports imgix, cloudinary and akamai image optimizers. But in my opinion it would be very useful to not only support image optimization services, but also support the image optimization build into Storyblok itself.

### Describe the solution you'd like

Add an image loader for Storyblok to Next.js. This way any project using Storyblok as a CMS does not need a custom image loader to be written.

### Describe alternatives you've considered

We have used custom image loaders for Storyblok projects thusfar. The problem is that we often have new projects using a custom image loader, so we need to copy over the code from one project to another. While it does not take that much time, it would be much more efficient for us and anyone else using Storyblok as a CMS to just change the `images.loader` to a value of `storyblok`.

**Comment by balazsorban44** [comment]
You can use any loader as described in the docs: https://nextjs.org/docs/api-reference/next/image#loader

We don't plan to add built-in loaders anymore. The ones from https://nextjs.org/docs/api-reference/next/legacy/image#built-in-loaders are deprecated.
